corehq/apps/app_manager/views/releases.py

```diff
@@ -250,3 +257,85 @@ def update_build_comment(request, domain, app_id):
     build.build_comment = request.POST.get('comment')
     build.save()
     return json_response({'status': 'success'})
+
+
+def _get_app_diff_files(app):
```

Comment:
Why not just add app.json to the View Source Files page? It wouldn't be harmful there.